<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
background: white;
color: blue;
font-size: 20px;
border: none;
border-bottom:  1px solid black;
padding: 5px;
}

#mainFrame {
border: none;
background: #53AEF0;
}

#loginFrame {
background: #CECECE;
border: 1px solid gray;
padding: 10px;
border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Chat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="chatPage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="2">
            <widget class="QPushButton" name="sayButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/envoi.png</normaloff>:/images/envoi.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="sayLineEdit"/>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QListWidget" name="userListWidget"/>
           </item>
           <item row="1" column="0">
            <widget class="QTextEdit" name="roomTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Liste des utilisateurs</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="loginPage">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QFrame" name="loginFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="spacing">
               <number>20</number>
              </property>
              <item row="0" column="4">
               <widget class="QRadioButton" name="btn_bug">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/bug.png</normaloff>:/images/bug.png</iconset>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Pseudo</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="btn_bitcoin">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/bitcoin.png</normaloff>:/images/bitcoin.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="btn_fusee">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/fusee.png</normaloff>:/images/fusee.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Adresse serveur</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QRadioButton" name="btn_android">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/android.png</normaloff>:/images/android.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QRadioButton" name="btn_apple">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/apple.png</normaloff>:/images/apple.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QRadioButton" name="btn_github">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/github.png</normaloff>:/images/github.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="3" colspan="3">
               <widget class="QLineEdit" name="serverLineEdit">
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>ex : 192.168.1.38</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3" colspan="3">
               <widget class="QLineEdit" name="userLineEdit">
                <property name="placeholderText">
                 <string>ex : superPseudo</string>
                </property>
               </widget>
              </item>
              <item row="3" column="4" colspan="2">
               <widget class="QPushButton" name="loginButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Connexion</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>223</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>165</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>25</width>
               <height>150</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>223</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serverLineEdit</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>roomTextEdit</tabstop>
  <tabstop>userListWidget</tabstop>
  <tabstop>sayLineEdit</tabstop>
  <tabstop>sayButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sayLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sayButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>userLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
